### PR TITLE
added bio

### PR DESCRIPTION
**Pull request by MahmoudAbdelRahman**

### CODE REVIEW

File: content/author/mahmoud-abdelrahman/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+authors:
+- Mahmoud AbdelRahman
```
Comment (adChong) [inline content/author/mahmoud-abdelrahman/_index.md]: authors should follow the naming convention of your folder mahmoud-abdelrahman

File: content/author/mahmoud-abdelrahman/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - 
+email: "mahmoud[at]u.nus.edu"
+interests:
+- Building performance modelling and simulation.
+- Spatio-temporal data analytics.
+- Computational Intelligence.
+name: Mahmoud AbdelRahman
+organizations:
+- name: National University of Singapore
+  url: "https://blog.nus.edu.sg/adrianchong/about/"
```
Comment (adChong) [inline content/author/mahmoud-abdelrahman/_index.md]: url: ""

**Review by adChong — COMMENTED** [review]
remove space in front of avatar.jpg

**Comment by MahmoudAbdelRahman** [comment]
@adChong  I've solved the problems.

**Pull request merged**